SwaglyricsBot/general_commands.py

```diff
@@ -142,7 +142,7 @@ def artists_to_string(artists):
     @staticmethod
     def get_lyrics(song, artist):
         """
-        Fetches lyrics using swaglyrics library
+        Fetches lyrics using the Swaglyrics library
```

Comment:
the is ok but swaglyrics works because it's `import swaglyrics`, while the application is SwagLyrics for Spotify